src/kata2.booleanToWord.js

```diff
@@ -1,3 +1,14 @@
-const booleanToWord = (boolean) => {};
+const booleanToWord = (boolean) => {
+  if (boolean === true) {
+    return "Yes";
+  }
+  if (boolean === false) {
+    return "No";
+  }
+  return "not a boolean";
```

Comment:
Nice to see you thinking about cases other than passing in boolean true or false! Edge cases are always worth thinking about as it makes your code more robust and less prone to bugs, but for this simple kata we can assume it will always be a proper boolean. In that case, this can be replaced simply with a ternary operator:
```
  return boolean === true ? "Yes" : "No";
```